scripts/setup_ubuntu.sh

```diff
@@ -110,10 +110,15 @@ apt update
 # Install packages required by the requirements tool
 apt install --yes --quiet python3-yaml python3-packaging
 
+if [ "$VERSION" == "18" ]; then
+  PKG_SYS_SUFFIX="-18.04"
+else
+  PKG_SYS_SUFFIX=""
+fi
 # Use the requirements tool to install apt packages with: apt-get satisfy --allow-downgrades --yes --quiet
-"$REPO_ROOT"/scripts/requirements install -a--allow-downgrades -a--yes -a--quiet -l apt -l apt/dev -f apt
+"$REPO_ROOT"/scripts/requirements install -a--allow-downgrades -a--yes -a--quiet -l apt -l apt/dev -p "apt$PKG_SYS_SUFFIX"
```

Comment:
I suppose `argparse` is okay with `-a--allow-downgrades` but would it be possible to be a little more explicit with `--arg=--allow-downgrades`?